### DISCUSSION TITLE
Kimai with active directory (Ldap) - users that already exists are with two passwords

### DISCUSSION THREAD

**Discussion by guicomoti1** [kimai/kimai#3143] [Questions & Answers]
users that already exists are with two passwords. one password is from the user created directly by Kimai and the other password is from active directory. new users are normal with one password.

i would like to know if there is a way do "force" users use only the passaword by Ldap ?

**Comment by kevinpapst** [comment]
See "Password handling" in https://www.kimai.org/documentation/ldap.html#user-synchronization
